### PR TITLE
Fix pagination issue (showing one page only)

### LINKED ISSUE #41: Pagination for user list only shows one page
In the list of users (and possibly list of errors) views, the pagination controls will only show that there is a single page even when the response has a count that would overflow into more than one page. E.g., this is the otp users API response and corresponding list/paginated view.

The limit per page is 10 (and total count is 39), but the pagination controls only show one page.

image

image

### PR DESCRIPTION
**Pull request by landonreed**
This PR fixes #41.

To test: 
- visit http://localhost:3000/manage?type=admin or deployed vercel link below
- Create more than ten users
- View multiple pages in page controls

### CODE REVIEW

**Review by miles-grant-ibigroup — APPROVED** [review]
`data.data` oy vey. But looks good!

File: components/PageControls.js [hunk 1]
```diff
@@ -3,9 +3,9 @@ import { Button, Pagination } from 'react-bootstrap'
 
 function PageControls ({ limit, offset, result, setOffset, showSkipButtons = false, url }) {
   const { data: swrData = {}, error, isValidating, mutate } = result
-  const { records } = swrData
-  const hasRecords = records && records.data && records.data.length > 0
-  const lastOffset = hasRecords ? records.total - (records.total % limit) : 0
+  const { data: responseList } = swrData
+  const hasRecords = responseList && responseList.data && responseList.data.length > 0
```
Comment (binh-dam-ibigroup) [inline components/PageControls.js:7]: I think we started using this syntax haven't we?
```suggestion
  const hasRecords = responseList?.data?.length > 0
```

File: components/PageControls.js [hunk 2]
```diff
@@ -20,7 +20,7 @@ function PageControls ({ limit, offset, result, setOffset, showSkipButtons = fal
           </Button>
           {hasRecords &&
             <span>
-              Showing {records.offset + 1} - {records.offset + records.data.length} of {records.total}
+              Showing {responseList.offset + 1} - {responseList.offset + responseList.data.length} of {responseList.total}
```
Comment (binh-dam-ibigroup) [inline components/PageControls.js:23]: Fields `total` could be extracted by destructuring assignment.

**Review by binh-dam-ibigroup — APPROVED** [review]
LGTM with some minor suggestions.

**Pull request merged**